image/image_test.go

```diff
@@ -1495,41 +1495,41 @@ func (s *imageSuite) TestSetupSeedWithBaseLegacySnap(c *C) {
 	c.Check(s.curSnaps[0], HasLen, 0)
 	c.Check(s.curSnaps[1], DeepEquals, []*store.CurrentSnap{
```

Comment:
Expected snaps in this list is now ordered for test consistency (we use a map instead of a slice for the list of local snaps), and the only reason there is a change to this file.

Reply:
Nice